### DISCUSSION TITLE
Add Plex Media Server LIBRARY  as LOCAL! source

### DISCUSSION THREAD

**Discussion by marcschh** [music-assistant/support#1197] [Feature requests and ideas]
Hi,

somehow PLEX can be used without the cloud stuff. From my understanding it is an DLNA Source. I have enabled this but I am not able to connect MusicAssistant to this source. Would be great if this is possible.

thanks
Marc

**Comment by marcelveldt** [comment]
Music Assistant can play to DLNA players but not read DLNA sources.

Plex just uses the MyPlex login, your data is still local.

**Reply by marcschh** [reply]
thanks for clarifing. 
1. but if internet is down it cannot be played. or is the login only for setting it up?
2. I tried it to set it up with the myplex login and the token. but the documentation is missing where to find the "The name of the server (as shown in the interface)" I tired some of the names I thought but it always failed

**Reply by marcelveldt** [reply]
1) login only
2) its just the name of your server (in the left bar)

Maybe try out next beta because we changed some stuff to make it way easier to set up Plex. New beta should arrive today or tomorrow.
